mobile/lib/models/user.model.dart

```diff
@@ -0,0 +1,6 @@
+class User {
+  String name;
+  String email;
+
+  User({required this.name, required this.email});
```

Comment:
I think it's better if we keep separate `firstName` and `lastName`.
```suggestion
  String firstName;
  String lastName;
  String email;

  User({required this.firstName, required this.lastName, required this.email});
```

Reply:
You're completely right 👍